go.mod

```diff
@@ -6,28 +6,29 @@ require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/dustin/go-humanize v1.0.0
 	github.com/efritz/pentimento v0.0.0-20190429011147-ade47d831101
-	github.com/fatih/color v1.9.0
-	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/google/go-cmp v0.4.1
-	github.com/hashicorp/go-multierror v1.1.0
+	github.com/fatih/color v1.11.0
+	github.com/ghodss/yaml v1.0.0
+	github.com/google/go-cmp v0.5.5
+	github.com/hashicorp/go-multierror v1.1.1
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
-	github.com/mattn/go-colorable v0.1.6 // indirect
 	github.com/mattn/go-isatty v0.0.12
-	github.com/mattn/go-runewidth v0.0.9 // indirect
 	github.com/neelance/parallel v0.0.0-20160708114440-4de9ce63d14c
 	github.com/olekukonko/tablewriter v0.0.4 // indirect
 	github.com/pkg/browser v0.0.0-20180916011732-0a3d74bf9ce4
 	github.com/pkg/errors v0.9.1
+	github.com/rivo/uniseg v0.2.0 // indirect
 	github.com/segmentio/textio v1.2.0
-	github.com/sourcegraph/codeintelutils v0.0.0-20200706141440-54ddac67b5b6
+	github.com/sourcegraph/codeintelutils v0.0.0-20210303211619-e3653453912b
 	github.com/sourcegraph/jsonx v0.0.0-20200629203448-1a936bd500cf
+	github.com/sourcegraph/sourcegraph/lib v0.0.0-20210622093026-6470ff817296
 	github.com/ssor/bom v0.0.0-20170718123548-6386211fdfcf // indirect
 	github.com/xeipuuv/gojsonpointer v0.0.0-20190905194746-02993c407bfb // indirect
 	github.com/xeipuuv/gojsonschema v1.2.0
-	golang.org/x/net v0.0.0-20200625001655-4c5254603344
-	golang.org/x/sys v0.0.0-20200622214017-ed371f2e16b4 // indirect
-	gopkg.in/yaml.v2 v2.3.0 // indirect
+	golang.org/x/net v0.0.0-20210614182718-04defd469f4e
 	jaytaylor.com/html2text v0.0.0-20200412013138-3577fbdbcff7
 )
 
 replace github.com/gosuri/uilive v0.0.4 => github.com/mrnugget/uilive v0.0.4-fix-escape
+
+// See: https://github.com/ghodss/yaml/pull/65
+replace github.com/ghodss/yaml => github.com/sourcegraph/yaml v1.0.1-0.20200714132230-56936252f152
```

Comment:
I'm so confused where this came from? I didn't do this. Maybe `go mod` magic did it via what is in sourcegraph/lib? Just FYI if you see some funky regression related to yaml.

Reply:
argh I have a really old base revision. my bad. I think I ddi this while fixing something with a `git checkout origin/main`.